docs/maven-example.md

```diff
@@ -0,0 +1,74 @@
+# Fabric8 Maven Plugin + Kompose:
+Lets deploy a Springboot Java application with Docker Compose file using Fabric8 Maven Plugin to Kubernetes or OpenShift.
+##### Requirements
+* Linux or MacOS or Windows
+* JDK 1.7+ - [JDK Quick Installation Guide](http://openjdk.java.net/install/)
+* Maven 3.x+ - [Maven Installation Guide](https://maven.apache.org/install.html) 
+* Kompose
+
```

Comment:
Perhaps add a previous step here saying that you must install Maven, here are the links to install to windows, etc. Add some commands here, etc.

Reply:
`+* Maven 3.x+ - [Maven Installation Guide]` here we have given the link for installation guide. Isn't it sufficient? 
In that way it would stay consistent with `Java` installation link in the same section.

Reply:
Alright. :+1: Let's just keep it like this for-now.